pt/cheatsheet-supervised-learning.md

```diff
@@ -1,567 +1,567 @@
 
-&#10230;
+&#10230; Dicas de Aprendizado Supervisionado
 
 <br>
 
 **2. Introduction to Supervised Learning**
 
-&#10230;
+&#10230; Introdução ao Aprendizado Supervisionado
 
 <br>
 
 **3. Given a set of data points {x(1),...,x(m)} associated to a set of outcomes {y(1),...,y(m)}, we want to build a classifier that learns how to predict y from x.**
 
-&#10230;
+&#10230; Dado um conjunto de dados {x(1),...,x(m) associados a um conjunto de resultados {y(1),...,y(m)}, nós queremos construir um classificador que aprende como predizer y baseado em x.
 
 <br>
 
 **4. Type of prediction ― The different types of predictive models are summed up in the table below:**
 
-&#10230;
+&#10230; Tipos de predição - Os diferentes tipos de modelo de predição estão resumidos na tabela abaixo:
 
 <br>
 
 **5. [Regression, Classifier, Outcome, Examples]**
 
-&#10230;
+&#10230; [Regressão, Classificador, Resultado, Exemplos]
 
 <br>
 
 **6. [Continuous, Class, Linear regression, Logistic regression, SVM, Naive Bayes]**
 
-&#10230;
+&#10230; [Contínuo, Classe, Regressão Linear, Regressão Logística, Máquina de Vetores de Suporte, Naive Bayes]
 
 <br>
 
 **7. Type of model ― The different models are summed up in the table below:**
 
-&#10230;
+&#10230; Tipos de modelo - Os diferentes modelos estão resumidos na tabela abaixo:
 
 <br>
 
 **8. [Discriminative model, Generative model, Goal, What's learned, Illustration, Examples]**
 
-&#10230;
+&#10230; [Modelo discriminativo, Modelo Generativo, Objetivo, O que é aprendido, Ilustração, Exemplos]
 
 <br>
 
 **9. [Directly estimate P(y|x), Estimate P(x|y) to then deduce P(y|x), Decision boundary,  	Probability distributions of the data, Regressions, SVMs, GDA, Naive Bayes]**
 
-&#10230;
+&#10230; [Estimar diretamente P(y|x), Estimar P(x|y) para daí deduzir P(x|y), Fronteira de decisão, Probabilidade da distribuição dos dados, Regressões, MVSs, GDA, Naive Bayes] 
 
 <br>
 
 **10. Notations and general concepts**
 
-&#10230;
+&#10230; Notações e conceitos gerais
 
 <br>
 
 **11. Hypothesis ― The hypothesis is noted hθ and is the model that we choose. For a given input data x(i) the model prediction output is hθ(x(i)).**
 
-&#10230;
+&#10230; Hipótese - A hipótese é denominada hθ e é o modelo que escolhemos. Para um determinado dado de entrada x(i) o resultado do modelo de predição é hθ(x(i)).
 
 <br>
 
 **12. Loss function ― A loss function is a function L:(z,y)∈R×Y⟼L(z,y)∈R that takes as inputs the predicted value z corresponding to the real data value y and outputs how different they are. The common loss functions are summed up in the table below:**
 
-&#10230;
+&#10230; Função de perda - A função de perda é definida como L:(z,y)∈R×Y⟼L(z,y)∈R que recebe como entradas o valor z previsto correspondente ao valor real y e retorna o quão diferente eles são.
 
 <br>
 
 **13. [Least squared error, Logistic loss, Hinge loss, Cross-entropy]**
 
-&#10230;
+&#10230; [Erro quadrático mínimo, Perda logística, Hinge loss, Entropia cruzada]
```

Comment:
I think _Perda de Hinge_ or _Perda de Articulação_ here it's more suitable (as you already used in the ln 279.